README.md

```diff
@@ -11,19 +17,76 @@ I) Code quality
 II) Simplicity
 - Interpreter only
 - Provide no support for floating point operations (initially)
-- Support only Wasm binary encoding as an input (no support for WAT)
+- Support only WebAssembly binary encoding as an input (no support for WAT)
 
 III) Conformance
-- Should pass the official Wasm test suite
+- Should pass the official WebAssembly test suite
 
 IV) First class support for determistic applications (*blockchain*)
-- Support the bigint API
+- Support an efficient big integer API (256-bit and perhaps 384-bit)
 - Support runtime metering in the interpreter
+- Support enforcing a call depth bound
+- Further restrictions of complexity (e.g. number of locals, number of function parameters, number of labels, etc.)
+
+## Building and using
+
+Fizzy uses CMake as a build system:
+```sh
+$ mkdir build && cd build
+$ cmake ..
+$ cmake --build .
+```
+
+This will build Fizzy as a library and since there is no public API
+(the so called *embedder API* in WebAssembly) yet, this is not very useful.
+
+Building with the `FIZZY_TESTING` option will output a few useful utilities:
+
+```sh
+$ mkdir build && cd build
+$ cmake -DFIZZY_TESTING=ON ..
+$ cmake --build .
+```
+
+These utilities are as follows:
+
+### fizzy-bench
 
-## Testing
+This can be used to run benchmarks available in the `test/benchmarks` directory.
+Read [this guide](./test/benchmarks/README.md) for a short introduction.
 
-To read about testing see [fizzy-spectests](./test/spectests/README.md).
+### fizzy-spectests
+
+Fizzy is capable of executing the official WebAssembly tests.
+Follow [this guide](./test/spectests/README.md) for using the tool.
+
+### fizzy-unittests
+
+This is the unit tests suite of Fizzy.
+
+## Maintainers
+
+- Alex Beregszaszi [@axic]
+- Andrei Maiboroda [@gumb0]
+- Paweł Bylica [@chfast]
 
 ## License
 
-Apache 2.0
+[![license badge]][Apache License, Version 2.0]
+
+Licensed under the [Apache License, Version 2.0].
+
+[webassembly]: https://webassembly.org/
+[standard readme]: https://github.com/RichardLitt/standard-readme
+[circleci]: https://circleci.com/gh/wasmx/fizzy/tree/master
+[codecov]: https://codecov.io/gh/wasmx/fizzy/
+[Apache License, Version 2.0]: LICENSE
+[@axic]: https://github.com/axic
+[@chfast]: https://github.com/chfast
+[@gumb0]: https://github.com/gumb0
+
+[webassembly badge]: https://img.shields.io/badge/WebAssembly-Engine-informational.svg?logo=webassembly
+[readme style standard badge]: https://img.shields.io/badge/readme%20style-standard-brightgreen.svg
+[circleci badge]: https://img.shields.io/circleci/project/github/wasmx/fizzy/master.svg?logo=circleci
+[codecov badge]: https://img.shields.io/codecov/c/github/wasmx/fizzy.svg?logo=codecov
```

Comment:
this one looks to me like this 
![image](https://user-images.githubusercontent.com/1863135/80128204-a00c3f80-8595-11ea-8c8d-867c2bd30175.png)

Reply:
and now it's fine

Reply:
I think github is sometimes down, something is broken.